jspm.js

```diff
@@ -26,7 +27,15 @@ var jspmCLI = new Liftoff({
     '.json': null
   }
 });
-jspmCLI.launch({}, function(env) {
+
+var args = process.argv.slice(2);
+
+var cwdArgIndex = args.indexOf('--cwd');
+if (cwdArgIndex > -1) {
+  env.setCwd(args[cwdArgIndex + 1]);
+}
+
+jspmCLI.launch({cwd: env.cwd()}, function(env) {
```

Comment:
👍  just space this - `{ cwd: env.cwd() }`.

Did you test this with both globally and locally installed jspm?